util/src/main/java/com/google/cloud/hadoop/util/EntriesCredentialConfiguration.java

```diff
@@ -57,6 +61,13 @@
    * service accounts. The default is to use a service account.
    */
   public static final String ENABLE_SERVICE_ACCOUNTS_SUFFIX = ".auth.service.account.enable";
+
+  // TODO Document the parameters. Additional documentation in gs-defaults.xml, toString
```

Comment:
gs-defaults.xml -> gcs-core-default.xml

Reply:
Removed the TODO. This was already done.